packages/query/src/index.test.ts

```diff
@@ -0,0 +1,36 @@
+import { it, expect, describe } from 'vitest';
+import { builder } from './index';
+import {
+  GeneratorOptions,
+  GeneratorVerbOptions,
+  NormalizedOverrideOutput,
+} from '@orval/core';
+
+describe('throws when trying to use named parameters with vue-query client', () => {
```

Comment:
Considering we're not permitting the use of `useNamedParameters` with `vue-query` do we still get value from adding `samples/vue-query-useNamedParameters` in this PR?

Reply:
Yeah good point, I removed the sample to avoid confusion in users. And added some comments for future developments: https://github.com/Maxim-Mazurok/orval/blob/13ec0680778217fd54608903e34d17be6b17fdee/scripts/update-samples.mjs#L17-L31